lib.js

```diff
@@ -59,9 +59,9 @@ const generateTestcaseResult = (testcase) => {
   if (!isTestcaseSuccess(testcase)) {
     let errorLines = '';
     if (testcase.failure.join) {
-      errorLines = testcase.failure.join(EOL).split(EOL).join(EOL + '\t');
+      errorLines = testcase.failure.map(f => f._).join(EOL).split(EOL).join(EOL + '\t');
     } else {
-      errorLines = testcase.failure;
+      errorLines = testcase.failure.map(f => f._);
```

Comment:
Just curious why this new map is required.

Reply:
`testcase.failure` is an object that includes other properties on it such as `type`. I needed this to get the actual failure message. 
Otherwise I ended up with:
```
 ✖ [It] Common tests has all the control-plane nodes running (0.001326583)
        [object Object]
```

Note: This might not always be the case as I'm using Ginkgo to generate the JUnit reports I'm working with. If this is a quirk to Ginkgo I'm happy to remove this and look for another solution.

Reply:
I see. That could be because the format of the failure tag can be different. It can be either of these:
```
<failure message="Assertion error message" type="AssertionError">
     Call stack printed here 
</failure> 

<failure type="NotEnoughFoo">details about failure</failure>

<failure message="PROGRAM.cbl:2 Use a program name that matches the source file name" type="WARNING">
  WARNING: Use a program name that matches the source file name
  Category: COBOL Code Review – Naming Conventions
  File: /project/PROGRAM.cbl
  Line: 2
</failure>
```

It's only in cases where there is no message using _ would work. In xml2js _ means child textnode value. So the below can be used to cover all 3 cases:

```suggestion
      errorLines = testcase.failure.map(f => f['$'] && f['$'].message ? f['$'].message : f['_'])
        .join(EOL).split(EOL).join(EOL + '\t');
    } else {
      errorLines = testcase.failure['$'] && testcase.failure['$'].message ?
        testcase.failure['$'].message : testcase.failure['_'];
```

Reply:
Updated 👍